src/server/handler.js

```diff
@@ -1,113 +1,144 @@
-const fs = require("fs");
-const path = require("path");
-const getData = require("../queries/getData");
-const postData = require("../queries/postData");
-const qs = require("querystring");
+const fs = require('fs');
+const path = require('path');
+const qs = require('querystring');
+const bcrypt = require('bcrypt');
+const { sign, verify } = require('jsonwebtoken');
+const postData = require('../queries/postData');
+const { getData, getLoginData } = require('../queries/getData');
+require('env2')('../../config.env');
```

Comment:
This should be require in the root file instead of in every place you want to use it.